pkg/rpm_pfg.bzl

```diff
@@ -791,21 +794,30 @@ pkg_rpm = rule(
         # funny if it's not provided.  The contents of the RPM are believed to
         # be set as expected, though.
         "architecture": attr.string(
-            doc = """Package architecture.
+            doc = """Host architecture.
 
             This currently sets the `BuildArch` tag, which influences the output
             architecture of the package.
 
             Typically, `BuildArch` only needs to be set when the package is
```

Comment:
You might as well make the exact  behavior and use clear and tie it back to Bazel definitions when the terms overlap. The points to make are

- If the package is really architecture independent, say "noarch"
- Otherwise specify a CPU architecture of the target platform your are building for.
- If you say nothing, it will default to the CPU arch of your execution platform.

Note that when cross compiling the target and execution platforms may not match.

Reply:
@aiuto @ogalbxela  when cross-compiling, target/exec may differ, but the logical default value would be the target.

When not cross-compiling, the target and exec match, so the logical default value would be the target.

Is there interest in setting the default to the target rather than exec platform?  Or a risk of breaking change in the default ?